code/modules/vehicles/collision.dm

```diff
@@ -0,0 +1,76 @@
+/atom/proc/tank_collision(obj/vehicle/roadtrafficaccident, facing, turf/T, turf/temp)

```

Comment:
If you have this proc you need to deal with all those CanPass procs doing similar things regarding collision damage